### PR TITLE
Hotfix/v2024.11.23

### PR DESCRIPTION
**Pull request by surapuramakhil**

### CODE REVIEW

File: src/ai_hawk/job_manager.py [hunk 1]
```diff
@@ -13,6 +14,7 @@
 
 from ai_hawk.linkedIn_easy_applier import AIHawkEasyApplier
 from config import JOB_MAX_APPLICATIONS, JOB_MIN_APPLICATIONS, MINIMUM_WAIT_TIME_IN_SECONDS
+import job
```
Comment (49Simon) [inline src/ai_hawk/job_manager.py:17]: Unused import should be removed. Can you also remove unused import in line 8?

File: src/ai_hawk/job_manager.py [hunk 2]
```diff
@@ -239,7 +241,7 @@ def start_applying(self):
                     time.sleep(sleep_time)
                 page_sleep += 1
 
-    def get_jobs_from_page(self):
+    def get_jobs_from_page(self, scroll=False):
```
Comment (49Simon) [inline src/ai_hawk/job_manager.py:244]: I'm curious, why do you have the default value as False?

**Review by 49Simon — DISMISSED** [review]
Tested it and its applying to jobs. Once these issues are addressed it can be merged as patch.

File: src/ai_hawk/job_manager.py (hunk 2 above)
Comment (surapuramakhil) [reply src/ai_hawk/job_manager.py:244]: to use it only when required.

**Comment by surapuramakhil** [comment]
@49Simon reviews typically happen when code goes to release / hotfix branches. This is mostly for testing purposes, more like release branch should be working on at least two others.

**Comment by 49Simon** [comment]
You are merging new code into the `main` branch as a critical patch, skipping the usual release process. It’s still important to fully test and review the code.

**Comment by ariaushani** [comment]
Has this issue been fixed? What's the latest working branch?

**Pull request merged**